dash_bio/metadata.json

```diff
@@ -1763,6 +1763,135 @@
       }
     }
   },
+  "src/lib/components/Molecule2dViewer.react.js": {
+    "description": "The Molecule2dViewer component is used to render structural\nformulae of molecules.\nRead more about the component here:\nhttps://github.com/Autodesk/molecule-2d-for-react",
+    "displayName": "Molecule2dViewer",
+    "methods": [
+      {
+        "name": "onChangeSelection",
+        "docblock": null,
+        "modifiers": [],
+        "params": [
+          {
+            "name": "selectedAtomIds",
+            "type": null
+          }
+        ],
+        "returns": null
+      }
+    ],
+    "props": {
+      "id": {
+        "type": {
+          "name": "string"
+        },
+        "required": false,
+        "description": "The ID used to identify this component in callbacks"
+      },
+      "setProps": {
+        "type": {
+          "name": "func"
+        },
+        "required": false,
+        "description": "Dash-assigned callback that should be called whenever properties change"
```

Comment:
This description should end with a period to be consistent with the following ones (below). Regarding its content, I'm not sure about writing the "that should be" part... If I understand correctly, this callback gets called whenever properties change, I wouldn't expect the end-user to have to do it 'manually' ("should be" sounds like an instruction in this context). Please enlighten me :)

Reply:
This is just added from the cookiecutter boilerplate (which was used to generate this repository), and is consequently part of all of the components: https://github.com/plotly/dash-component-boilerplate/blob/64406a8ecec85da56b40e930d48fa66d1a3b309c/%7B%7Bcookiecutter.project_shortname%7D%7D/src/lib/components/%7B%7Bcookiecutter.component_name%7D%7D.react.js#L40

`setProps` is actually not necessarily called every time properties change; if you look in the code for the components, the author _does_ have to "manually" add in a call to `setProps`. Using this function allows for exchange of information between the React component and the Dash app.

I agree that it could be worded better (e.g., "Dash-assigned callback that should be called whenever information regarding property changes needs to be exchanged with Dash."), but "should be called" is correct here as far as I can tell.

Reply:
Ok, thanks for the details and for pointing me to the source! I'll poke @Marc-Andre-Rivet on this then ;)